recipes/easy_profiler/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conans import ConanFile, tools, CMake
+import os
+import glob
+
+
+class easy_profilerConan(ConanFile):
+    name = "easy_profiler"
+    description = "Lightweight profiler library for c++"
+    license = "BSD"
```

Comment:
```suggestion
    license = "MIT"
```

https://github.com/yse/easy_profiler/blob/v2.1.0/LICENSE